core/workspace_comment/workspace_comment_svg.js

```diff
@@ -503,3 +516,7 @@ Blockly.WorkspaceCommentSvg.prototype.setDeleteStyle = function(enable) {
         /** @type {!Element} */ (this.svgGroup_), 'blocklyDraggingDelete');
   }
 };
+
+Blockly.WorkspaceCommentSvg.prototype.setAutoLayout = function() {
```

Comment:
I feel like I'm missing something. I don't see where this would get called. Is there another change that's making this part of the bubble dragger path?

Reply:
The change to make this use the bubble dragger is right here: https://github.com/google/blockly/pull/1661/files#diff-2d76f92ba68d3e43c2c00ddf993751deR148

That's where I'm calling `gesture.handleBubbleStart(e, this);`

The `setAutoLayout` part is from when I added the bubble dragger initially, and I haven't yet sorted out if that's realy part of the API that I want to keep around.  But it's called from the bubble dragger.